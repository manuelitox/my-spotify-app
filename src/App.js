import React, { Fragment } from 'react'
import { ThemeProvider } from 'styled-components'

import theme from './config/theme'
import Routers from './Routers'
import GlobalStyles from './GlobalStyles'

const App = () => (
  <ThemeProvider theme={ theme }>
    <Fragment>
      <Routers />
      <GlobalStyles /> 
    </Fragment>
  </ThemeProvider>
)

export default App;
