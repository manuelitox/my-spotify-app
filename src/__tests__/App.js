import React from 'react'
import { shallow } from 'enzyme'

import App from 'App'

describe('Component | App', () => {
  let app 
  beforeEach( () => {
    app = shallow(
      <App />
    )
  })
  it('should render an App', () => {
    expect(app).toMatchSnapshot()
  })
})