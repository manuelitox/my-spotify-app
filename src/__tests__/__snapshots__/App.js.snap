// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | App should render an App 1`] = `
<ThemeProvider
  theme={
    Object {
      "blueDark": "#262D45",
      "blueDarkText": "#A8A9C1",
      "blueLight": "#3D4EC9",
      "blueLightBorder": "#414C6F",
      "blueLightText": "#97A4FF",
      "maxWidth": "1200px",
      "offWhite": "#999999",
      "playIcon": "#41475d",
      "searchSpan": "#F7F7F7",
      "shadow": "rgba(0, 0, 0, .7)",
      "size": Object {
        "medium": "990px",
        "small": "760px",
      },
      "textFooter": "#385758",
    }
  }
>
  <Routers />
  <GlobalStyleComponent />
</ThemeProvider>
`;
